src/v2/Apps/Partner/PartnerApp.tsx

```diff
@@ -31,8 +31,15 @@ export const PartnerApp: React.FC<PartnerAppProps> = ({
     throw new HttpError(404)
   }
 
-  const isBlackOwned =
-    categories!.filter(c => c && c.name === "Black Owned").length > 0
+  const galleryBadges = ["Black Owned", "Women Owned"]
+
+  const eligibleCategories = (categories || []).filter(Boolean)
```

Comment:
Are you sure you don't want to call this `omg` or `lol` or something more fun!!

Reply:
Ya know....we tired that......and it got confusing......

`yas` was even floated around as a name